src/templates/plugin.routing.js

```diff
@@ -150,7 +150,7 @@ const VueInstanceProxy = function (targetFunction) {
       getRouteBaseName: this.getRouteBaseName,
       i18n: this.$i18n,
       localePath: this.localePath,
-      req: process.server ? this.$ssrContext.req : null,
+      req: process.server ? this.$ssrContext?.req ?? this.context?.ssrContext?.req : null,
```

Comment:
I wonder if you have found any reference/documentation for that `context` property somewhere. It seems public but not sure if it's official.

Reply:
No I have been looking for this as well in the official nuxt document, but have yet to find it. The closest I could find was https://nuxtjs.org/docs/2.x/internals-glossary/context but there is no mention of `ssrContext` in there.
I don't know if it is composition api specific either. Maybe @danielroe can shed some light on this?